### PR TITLE
New Settings: Show extension in full size view

### PR DESCRIPTION
**Pull request by dan437**
## **Description**
There are people who love to use the extension in the full size view. It's a bit time consuming to always click in the extension on three dots and then on Expand view. With this new setting the extension will open in the full size view after clicking on the extension icon (except if a 3rd party opens it, e.g. signing a Swap on Uniswap with MetaMask).

## **Screenshots/Recordings**
image

## **Testing**
- Open the extension
- Go to Settings -> Advanced
- Enable "Show extension in full size view"
- Click outside of the extension popup to close it
- Click on the MetaMask extension icon again, it will open in full size view

### CODE REVIEW

**Comment by darkwing** [comment]
Great feature addition!

**Comment by DDDDDanica** [comment]
LGTM ! Just need to update the snapshot for e2e test

One question: is it possible to append some e2e tests for this new feature ?

File: ui/pages/settings/advanced-tab/advanced-tab.container.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ import {
   setFeatureFlag,
   setShowFiatConversionOnTestnetsPreference,
   setShowTestNetworks,
+  setShowExtensionInFullSizeView,
```
Comment (digiwand) [inline ui/pages/settings/advanced-tab/advanced-tab.container.js:16]: [super nit] placement breaks alphabetical prop ordering here and other places above

**Review by digiwand — COMMENTED** [review]
Lgtm! +1 on @DDDDDanica's comment above regarding tests

**Review by DDDDDanica — APPROVED** [review]

**Review by digiwand — APPROVED** [review]
Lgtm!

**Pull request merged**